### ISSUE TITLE
Max number of rows and colums

### ISSUE THREAD

**Issue by picontrolit** [Chris--A/Keypad#20]
I have an old commodore keyboard with 8 rows and 16 cols. I adjusted the example sketch but keypresses are not detected at all.
I wonder if there is a maximum amount of cols and rows supported by the keypad library.
Especially this line from keypad.h seems to define a hard limitation.

#define MAPSIZE 10		// MAPSIZE is the number of rows (times 16 columns)

**Comment by jtornero** [comment]
Yes, it does limit the max number of keys. I made a PR some time ago that partially solves this see https://github.com/Chris--A/Keypad/pull/35 . But look sthis repo is abandoned.  I'm tryng to get to one of the authors though,

Providing the keymap is declared as an array of chars (AFAIK), the maximum number is 256 keys.


However
